step.yml

```diff
@@ -0,0 +1,85 @@
+title: |-
+  Simple Git Clone
+summary: |-
+  The step checks out the defined repository state without interfering with environmental variables.
```

Comment:
You can remove the ` without interfering with environmental variables` part of the sentence.